### DISCUSSION TITLE
Installing CloudLink on Replit

### DISCUSSION THREAD

**Discussion by FreezingChima** [MikeDev101/cloudlink#108] [Q&A] [answered]
Hi, I've been able to setup the cloudlink server on replit, but there is one thing that I do not understand. In the "Instanciating CloudLink" section of the docs ([https://hackmd.io/g6BogABhT6ux1GA2oqaOXA#Instanciating-CloudLink](https://hackmd.io/g6BogABhT6ux1GA2oqaOXA#Instanciating-CloudLink)) it tells me to paste some text into my "python file". Can someone explain to me what this so-called "python file" is?

Thanks in advance for any help.

**Comment by CST1229** [comment, marked as answer]
Probably the Python file that will be run to start the server (I think it should be called `main.py` on replit)
